pkg/localcommit/localcommit.go

```diff
@@ -64,6 +69,18 @@ func Get(projectDir string) (strfmt.UUID, error) {
 	return strfmt.UUID(commitID), nil
 }
 
+// GetCompatible returns the given project's commit ID in either the new format (commit file), or
+// the old format (activestate.yaml).
+func GetCompatible(proj projecter) (strfmt.UUID, error) {
+	if commitID, err := Get(proj.Dir()); err == nil {
+		return commitID, nil
+	} else if IsFileDoesNotExistError(err) {
+		return strfmt.UUID(proj.LegacyCommitID()), nil
+	} else {
+		return "", err
+	}
+}
```

Comment:
I don't think this belongs in the `localcommit` package. Its only concern should be the local commit file.

That said I realize the ship has kinda sailed with `AddToGitignore` also being implemented here. We should probably address that, but not as part of this PR. Still, we shouldn't add to the problem.

Reply:
I wasn't sure where this was supposed to go, so I threw it into a new `internal/runbits/commitid` package. I'm sure that's wrong, so please let me know what you prefer. Some clients will want `localcommit.Get()` because they require such a thing. Other clients will want a way to get a legacy commit ID or a new localcommit ID, so for now it's `commitid.GetCompatible()`.